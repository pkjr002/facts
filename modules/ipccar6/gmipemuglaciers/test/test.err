mv: cannot stat ‘*.nc’: No such file or directory
