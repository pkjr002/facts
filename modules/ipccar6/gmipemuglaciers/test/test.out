Initiating test in /scratch/rk509/test.1666639914...
Extracting data files...
Executing workflow...
python ipccar6_preprocess_gmipemuglaciers.py --pipeline_id glaciers-ipccar6-gmipemuglaciers-ssp585 --scenario ssp585 --baseyear 2005 --pyear_start 2020 --pyear_end 2100 --pyear_step 10 --model_driver FAIR
python ipccar6_fit_gmipemuglaciers.py --pipeline_id glaciers-ipccar6-gmipemuglaciers-ssp585
python ipccar6_project_gmipemuglaciers_dev.py --pipeline_id glaciers-ipccar6-gmipemuglaciers-ssp585 --nsamps 20000 --seed 4321
python ipccar6_postprocess_gmipemuglaciers_dev.py --pipeline_id glaciers-ipccar6-gmipemuglaciers-ssp585 --chunksize 662 --locationfile test/location_list.lst
Collecting output files...
